gapis/resolve/resource_meta.go

```diff
@@ -51,9 +41,32 @@ func (r *ResourceMetaResolvable) Resolve(ctx context.Context) (interface{}, erro
 		}
 		values[i] = val
 	}
-	result := &api.ResourceMeta{
-		IDMap:     res.resourceMap,
-		Resources: values,
+	return values, nil
+}
+
+// ResourceIDMap returns the ResourceMap at the given command.
+func ResourceIDMap(ctx context.Context, after *path.Command, r *path.ResolveConfig) (api.ResourceMap, error) {
+	resources, err := Resources(ctx, after.Capture, r)
+	if err != nil {
+		return nil, err
+	}
+
+	m := api.ResourceMap{}
+	created := map[string]*path.Command{}
+	for _, ty := range resources.Types {
+		for _, res := range ty.Resources {
+			if !after.IsAfter(res.Created) {
```

Comment:
Shouldn't this check that the `after` command is after the resource creation, to map only the resources which are created by the time of the `after` command?

```suggestion
			if after.IsAfter(res.Created) {
```

Reply:
Fixed.